docs/production/index.rst

```diff
@@ -49,7 +49,7 @@ The amount of effort you wish to expend ensuring that Lemur has good entropy to
 
 If you wish to generate more entropy for your system we would suggest you take a look at the following resources:
 
-- `WES-entropy-client <https://github.com/WhitewoodCrypto/WES-entropy-client>`_
+- `WES-entropy-client <https://github.com/Virginian/WES-entropy-client>`_
```

Comment:
maybe we should mark it as such, that this is clone of the presently deleted WhitewoodCrypto repo

Reply:
I've updated the pull request with 5d7ca85